### ISSUE TITLE
I hope I can use shovel at path block with any stage, to transform vanilla dirt path manually

### ISSUE THREAD

**Issue by MilkissWhite** [lebonq/Automatic-Path#45]
I using mod version 1.49 for MC 1.19.3 and it seems not a bug. I know there is config for 'automatic vanilla path' transform option. but I think there is no reason for prevent using shovel for manually transform.
now we can make vanilla path with shovel from dirt and grass both, so any path block stage should be work.

I decide to write this suggestion because sometimes I use shovel but it doesn't work, then I reconize why because it's a path block newly transformed, not a vanilla grass block. two looks very similar so I confuse sometimes. also making path will be easier and smooth with this suggest.

anyway, thank you for cool mod.

**Comment by SolacianAllie** [comment]
I agree, ive got this mod on my server and im trying to build permanent pathways and breaking, replacing then shoveling the paths is getting very boring, very quickly
